packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -111,8 +113,8 @@ export function colorMix(a: SpectrumColor, b: SpectrumColor, percent: number): `
   return `[color-mix(in srgb, ${parseColor(a)}, ${parseColor(b)} ${percent}%)]`;
 }
 
-export function linearGradient(angle: string, ...tokens: [SpectrumColor, number][]): string {
-  return `linear-gradient(${angle}, ${tokens.map(([color, stop]) => `${parseColor(color)} ${stop}%`)})`;
+export function linearGradient(angle: string, ...gradientTokens: [(keyof typeof tokens), (keyof typeof tokens)][]): string {
```

Comment:
unsure if making this function accept only tokens is ok. currently only used by the gradient buttons, but in the case someone else wanted to use it, it might need to accept color token as well?